docs/spectator/lang/go/usage.md

```diff
@@ -108,3 +108,12 @@ by providing one as the third parameter of the `Config` constructor.
 Use [spectator-go-runtime-metrics](https://github.com/Netflix/spectator-go-runtime-metrics). Follow
 instructions in the [README](https://github.com/Netflix/spectator-go-runtime-metrics) to enable
 collection.
+
+## Design Considerations - Reporting Intervals
+
+This client is stateless, and sends a UDP packet (or unixgram) to `spectatord` each time a meter is
+updated. If you are performing high-volume operations, on the order of tens-of-thousands or millions
+of operations per second, then you should pre-aggregate your metrics and report them at a cadence
+closer to the `spectatord` publish interval of 5 seconds, or the Atlas metrics granularity of 1 minute.
```

Comment:
Batching as a feature is not yet available in spectator-go, but it will be added before the end of the quarter.

Good point on the timing aspect, though!